install/stand_alone_registry.adoc

```diff
@@ -213,9 +213,9 @@ lb.example.com
 
 # host group for nodes, includes region info
 [nodes]
-master[1:3].example.com openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
-node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
-node2.example.com openshift_node_labels="{'region': 'primary', 'zone': 'west'}"
+master[1:3].example.com openshift_node_group_name='node-config-master-infra'
+node1.example.com openshift_node_group_name='node-config-compute'
+node1.example.com openshift_node_group_name='node-config-compute'
```

Comment:
@sdodson Updated node groups for these two stand-alone registry inventories seem OK?